src/main/java/org/fogbowcloud/saps/engine/core/dispatcher/restlet/resource/LinkResource.java

```diff
@@ -0,0 +1,80 @@
+package org.fogbowcloud.saps.engine.core.dispatcher.restlet.resource;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Properties;
+
+import org.apache.commons.httpclient.HttpStatus;
+import org.apache.log4j.Logger;
+import org.fogbowcloud.saps.engine.core.archiver.storage.AccessLink;
+import org.fogbowcloud.saps.engine.core.archiver.storage.PermanentStorage;
+import org.fogbowcloud.saps.engine.core.archiver.storage.PermanentStorageType;
+import org.fogbowcloud.saps.engine.core.archiver.storage.exceptions.TaskNotFoundException;
+import org.fogbowcloud.saps.engine.core.archiver.storage.nfs.NfsPermanentStorage;
+import org.fogbowcloud.saps.engine.core.archiver.storage.swift.SwiftPermanentStorage;
+import org.fogbowcloud.saps.engine.core.model.SapsImage;
+import org.fogbowcloud.saps.engine.utils.SapsPropertiesConstants;
+import org.restlet.data.Form;
+import org.restlet.data.MediaType;
+import org.restlet.representation.Representation;
+import org.restlet.representation.StringRepresentation;
+import org.restlet.resource.Get;
+import org.restlet.resource.ResourceException;
+
+import com.google.gson.Gson;
+
+public class LinkResource extends BaseResource {
+
+    private static final Logger LOGGER = Logger.getLogger(EmailResource.class);
+
+    private static final String REQUEST_ATTR_TASK_ID = "taskId";
+    
+    private final Gson gson = new Gson();;
+    
+    @Get
+    public Representation getTaskLinks(Representation representation) {
+        Form form = new Form(representation);
+
+        String userEmail = form.getFirstValue(UserResource.REQUEST_ATTR_USER_EMAIL, true);
+        String userPass = form.getFirstValue(UserResource.REQUEST_ATTR_USERPASS, true);
+
+        //FIXME I think that authenticateUser should throw an exception itself once
+        // the authentication process hasn't worked... - by @raonismaneoto
```

Comment:
besides I've said that, I guess @thiagomanel enjoys this current approach... let's see what he has to say about it

Reply:
agree. but, I have little to nothing context about this. I follow our opinion @raonismaneoto

Reply:
where does the need for this `userEmail.equals("anonymous")` come from? I did not understand this feature.

Reply:
This feature was already in the old code, but I believe it will not be used at any time (I believe).

Reply:
if you say so, I suggest you removing it.

Reply:
Removed